src/context/questionContext.js

```diff
@@ -7,6 +7,8 @@ const questionContext = createContext();
 const QuestionProvider = ({children}) => {
     const [ questionCategoryList, setQuestionCategoryList ] = useState([]);
     const [ resultInfo, setResultInfo ] = useState([]);
+  const [score, setScore] = useState(0);
+
 
```

Comment:
Suggestion: you can use useReducer here to manage state

Reply:
ok I will do it later